neo/core/container.py

```diff
@@ -18,27 +20,28 @@ def unique_objs(objs):
     using the "is" test.
     """
     seen = set()
-    return [obj for obj in objs
-            if id(obj) not in seen and not seen.add(id(obj))]
+    return [obj for obj in objs if id(obj) not in seen and not seen.add(id(obj))]
```

Comment:
please revert all white-space only changes, they make it harder to see the substantial changes.

Reply:
see: f9e1d290ad23ad0e393cbf86fd1fd56b38caeeb3